usaspending_api/awards/v2/views/awards.py

```diff
@@ -75,6 +75,16 @@ def _business_logic(self, request_dict: dict) -> dict:
         else:
             response_content = construct_assistance_response(request_dict)
 
+        response_content["disaster_emergency_fund_codes"] = [
+            {"code": row["disaster_emergency_fund__code"], "title": row["disaster_emergency_fund__title"]}
```

Comment:
There will be a number of non-CARES Act DEFC returned by this query. Perhaps add the DEFC "group" from the lookup table so the frontend can surface just the CARES Act DEFC

Reply:
good call